learn/advanced/updating.md

```diff
@@ -18,7 +18,12 @@ Before we begin, you need to **verify the version of Meilisearch that's compatib
 
 If Meilisearch launches successfully, use the [get version endpoint](/reference/api/version.md), note your `pkgVersion`, and [proceed to the next step](#proceed-according-to-your-database-version).
 
-<CodeSamples id="updating_guide_check_version" />
+<CodeSamples id="updating_guide_check_version_new_authorization_header" />
+
+::: note
+ If you're using v0.24 or below, use the `X-MEILI-API-KEY: apiKey` authorization header:
```

Comment:
Not sure if having a code sample in a note is a good idea

Reply:
I'm not sure I like the idea, but I'm also not sure I have any reason for not liking it? It doesn't look as bad as I expected on the preview.

Let's go with that. We can always change it if we don't like it once it's online or if we find a better solution.